src/test/java/com/google/cloud/tools/gradle/appengine/MultiModuleTestProject.java

```diff
@@ -65,6 +67,11 @@ public Project build() throws IOException {
       p.getPluginManager().apply(JavaPlugin.class);
       p.getPluginManager().apply(WarPlugin.class);
       p.getPluginManager().apply(AppEngineStandardPlugin.class);
+
+      DeployExtension deploy =
+          p.getExtensions().getByType(AppEngineStandardExtension.class).getDeploy();
+      deploy.setProjectId("project");
```

Comment:
yeah, this programmatic creation of the project isn't using a build file. And since we're resolving project/version a little earlier in the build, it will fail at configuration time if they're not set.